featuretools/tests/entityset_tests/test_es.py

```diff
@@ -433,7 +430,7 @@ def test_make_index_any_location(df):
                      make_index=True,
                      logical_types=logical_types,
                      dataframe=df)
-    if dd and isinstance(df, dd.DataFrame):
+    if not isinstance(df, pd.DataFrame):
```

Comment:
can use `es.dataframe_type` here if you'd like

Reply:
Good call! Updated.